src/blockchain/lending/helper.ts

```diff
@@ -0,0 +1,216 @@
+    LENDING_X_INCH_B_MARKET,
+    provider,
+    network
+  ) as Market
+  if (!xINCHbMarket) {
+    return Promise.reject(new Error('Could not create xINCHbMarket Contract'))
+  }
+  const xKNCaMarket = getContract(
+    LENDING_X_KNC_A_MARKET,
+    provider,
+    network
+  ) as Market
+  if (!xKNCaMarket) {
+    return Promise.reject(new Error('Could not create xKNCaMarket Contract'))
+  }
+  const xKNCbMarket = getContract(
+    LENDING_X_KNC_B_MARKET,
+    provider,
+    network
+  ) as Market
+  if (!xKNCbMarket) {
+    return Promise.reject(new Error('Could not create xKNCbMarket Contract'))
+  }
+  return {
+    xAAVEaMarket: xAAVEaMarket,
+    xAAVEbMarket: xAAVEbMarket,
+    xINCHaMarket: xINCHaMarket,
+    xINCHbMarket: xINCHbMarket,
+    xKNCaMarket: xKNCaMarket,
+    xKNCbMarket: xKNCbMarket,
+  }
+}
+
+export const getPricingContracts = async (provider: BaseProvider) => {
+  const network = await provider.getNetwork()
+  const xAAVEaPrice = getContract(
+    LENDING_X_AAVE_A_PRICE,
+    provider,
+    network
+  ) as XAAVEPrice
+  if (!xAAVEaPrice) {
+    return Promise.reject(new Error('Could not create xAAVEaPrice Contract'))
+  }
+  const xAAVEbPrice = getContract(
+    LENDING_X_AAVE_B_PRICE,
+    provider,
+    network
+  ) as XAAVEPrice
+  if (!xAAVEbPrice) {
+    return Promise.reject(new Error('Could not create xAAVEbPrice Contract'))
+  }
+  const xINCHaPrice = getContract(
+    LENDING_X_INCH_A_PRICE,
+    provider,
+    network
+  ) as XINCHPrice
+  if (!xINCHaPrice) {
+    return Promise.reject(new Error('Could not create xINCHaPrice Contract'))
+  }
+  const xINCHbPrice = getContract(
+    LENDING_X_INCH_B_PRICE,
+    provider,
+    network
+  ) as XINCHPrice
+  if (!xINCHbPrice) {
+    return Promise.reject(new Error('Could not create xINCHbPrice Contract'))
+  }
+  const xKNCaPrice = getContract(
+    LENDING_X_KNC_A_PRICE,
+    provider,
+    network
+  ) as XKNCPrice
+  if (!xKNCaPrice) {
+    return Promise.reject(new Error('Could not create xKNCaPrice Contract'))
+  }
+  const xKNCbPrice = getContract(
+    LENDING_X_KNC_B_PRICE,
+    provider,
+    network
+  ) as XKNCPrice
+  if (!xKNCbPrice) {
+    return Promise.reject(new Error('Could not create xKNCbPrice Contract'))
+  }
+  return {
+    xAAVEaPrice: xAAVEaPrice,
+    xAAVEbPrice: xAAVEbPrice,
+    xINCHaPrice: xINCHaPrice,
+    xINCHbPrice: xINCHbPrice,
+    xKNCaPrice: xKNCaPrice,
+    xKNCbPrice: xKNCbPrice,
+  }
+}
+
+export const getContracts = async (provider: BaseProvider) => {
+  const comptroller = await getComptroller(provider)
+  const liquidityPool = await getLiquidityPool(provider)
+  const lpt = await getLPT(provider)
+  const markets = await getMarkets(provider)
+  const prices = await getPricingContracts(provider)
+  const baseLendingProtocol = {
+    comptroller: comptroller,
+    liquidityPool: liquidityPool,
+    lpt: lpt,
+  }
+  return { ...baseLendingProtocol, ...markets, ...prices }
+}
+
+export const enum Markets {
```

Comment:
Rename it to something more meaningful like `LendingMarkets` for global usage?
```suggestion
export const enum LendingMarkets {
```